analyzers/tests/SonarAnalyzer.Test/SymbolicExecution/Roslyn/LoopDetectorTest.cs

```diff
@@ -312,8 +312,8 @@ public class LoopDetectorTest
             }
             _ = "After loop";                // Block 7
             """,
-            [2, 3, 4],
-            [1, 5, 6, 7]);
+            [2, 3, 4, 5],
```

Comment:
We need to add tests for some nested scenarios too. There's implementation for them.